spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -342,4 +349,30 @@ public AutoTimeProperties getAutotime() {
 
 	}
 
+    public static class System {
+
+        private final DiskSpace diskSpace = new DiskSpace();
```

Comment:
This should be named `Diskspace` to avoid the `-` in the configuration property.